tt_metal/hw/inc/wormhole/dev_mem_map.h

```diff
@@ -115,10 +115,10 @@
 #define MEM_IERISC_RESERVED1_SIZE 1024
 #define MEM_IERISC_MAILBOX_BASE (MEM_IERISC_RESERVED1 + MEM_IERISC_RESERVED1_SIZE)
 // TODO: reduce this when mailbox sizes are core type aware for some members (eg watcher/dprint)
-#define MEM_IERISC_MAILBOX_SIZE 3072
+#define MEM_IERISC_MAILBOX_SIZE 3104
 #define MEM_IERISC_MAILBOX_END (MEM_IERISC_MAILBOX_BASE + MEM_IERISC_MAILBOX_SIZE)
-#define MEM_IERISC_RESERVED2 4096
-#define MEM_IERISC_RESERVED2_SIZE 4096
+#define MEM_IERISC_RESERVED2 4128
```

Comment:
@pgkeller I had to modify the reserved space in the Idle ERISC mem map to get the new `mailboxes_t` struct to fit. Please let me know if this is okay.